### ISSUE TITLE
Top menu disappear when window is a bit smaller

### ISSUE THREAD

**Issue by jhagg** [gsliepen/ikistrap#8]
I can't get the top menu unless I change to fullscreen, I guess it thinks it is a mobile device, since I like smaller browser windows.

If I make the browser somewhat bigger I get the menu.

(My favorite browser width is currently 945 to 1326 pixels, depending on
device.)

Is it possible to control when it 'goes mobile'?

**Comment by gsliepen** [comment]
Yes. Edit templates/page.tmpl, and change the class "navbar-toggleable-md" to whatever size class you want it to toggle at. For example, "navbar-toggleable-sm" will cause it to toggle when the page width goes below 768 pixels. You can find more about how to control the navbar at https://v4-alpha.getbootstrap.com/components/navbar/, and some information about the responsive sizes that Bootstrap supports at https://v4-alpha.getbootstrap.com/layout/responsive-utilities/ and https://v4-alpha.getbootstrap.com/layout/overview/#responsive-breakpoints.
